src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx

```diff
@@ -304,8 +304,15 @@ export interface CircleColor {
   stops: string[][];
   type: string;
 }
+export interface PolygonColor {
+  property: string;
+  stops: string[][];
+  type: string;
+}
```

Comment:
tiny pick: this interface is similar to the one above it `CircleColor`.

Reply:
Ya. I personally think there should be a GenericColor interface but since the existing CircleColor was there I thought the best practice would be to be excplicit (e.g. Polygon...).